client/apps/shipping-label/index.js

```diff
@@ -30,14 +30,64 @@ import initializeLabelsState from 'woocommerce/woocommerce-services/lib/initiali
 import './style.scss';
 
 export default ( { order, accountSettings, packagesSettings, shippingLabelData, continents, context, items } ) => {
-
 	const orderId = order ? order.id : null;
-	const { storeOptions, formMeta, userMeta, formData } = accountSettings;
-	const packages = packagesSettings.formData;
-	const dimensionUnit = packagesSettings.storeOptions.dimension_unit;
-	const weightUnit = packagesSettings.storeOptions.weight_unit;
-	const packageSchema = packagesSettings.formSchema.custom ? packagesSettings.formSchema.custom.items : undefined;
-	const predefinedSchema = packagesSettings.formSchema.predefined;
+	let preLoadedState = false;
```

Comment:
(minor) simplify this to 
```suggestion
	const preLoadedState = undefined !== accountSettings;
```

Also, what does everyone think about prefixing booleans with a `has`/`is` prefix? 👀

Reply:
Both good suggestions! I went with `isPreloaded`